apps/assessment_registry/models.py

```diff
@@ -191,11 +191,14 @@ class AffectedGroupType(models.IntegerChoices):
 
     # Focus
     # -- Focus Sectors
-    focuses = ArrayField(models.IntegerField(choices=FocusType.choices, null=True, blank=True))
-    sectors = ArrayField(models.IntegerField(choices=SectorType.choices, null=True, blank=True))
-    protection_info_mgmts = ArrayField(models.IntegerField(choices=ProtectionInfoType.choices, null=True, blank=True))
+    focuses = ArrayField(models.IntegerField(choices=FocusType.choices, null=True, blank=True), blank=True)
```

Comment:
Maybe we should use default=list instead.
Basically to avoid storing both null and [] as an empty value in the database.